presto-main/src/test/java/com/facebook/presto/cost/TestCostCalculator.java

```diff
@@ -190,7 +190,8 @@ public void testTableScan()
     public void testProject()
     {
         TableScanNode tableScan = tableScan("ts", "orderkey");
-        PlanNode project = project("project", tableScan, new VariableReferenceExpression(Optional.empty(), "string", VARCHAR), new Cast(new SymbolReference("orderkey"), "VARCHAR"));
+        RowExpression cast = translator.translateAndOptimize(new Cast(new SymbolReference("orderkey"), "VARCHAR"), TypeProvider.viewOf(ImmutableMap.of("orderkey", BIGINT)));
```

Comment:
Why Optimized? Translate alone should be good enough?

Reply:
Yeah, updates. These are just tests though, shouldn't matter much.